lib/redux/actions/user_actions.dart

```diff
@@ -0,0 +1,81 @@
+
+import 'package:fusecash/redux/actions/error_actions.dart';
+import 'package:redux/redux.dart';
+import 'package:redux_thunk/redux_thunk.dart';
+import 'package:wallet_core/wallet_core.dart';
+
+class RestoreWalletSuccess {
```

Comment:
not so keen about passing mnemonic and pk here. redux actions is something viewable, for example in the Dapp we use it for debugging